moveit_experimental/moveit_jog_arm/src/collision_check_thread.cpp

```diff
@@ -92,12 +97,20 @@ void CollisionCheckThread::startMainLoop(JogArmShared& shared_variables)
       for (std::size_t i = 0; i < jts.position.size(); ++i)
         current_state.setJointPositions(jts.name[i], &jts.position[i]);
 
-      collision_result.clear();
       current_state.updateCollisionBodyTransforms();
 
       // Do a thread-safe distance-based collision detection
-      getLockedPlanningSceneRO()->checkCollision(collision_request, collision_result, current_state);
+      collision_result.clear();
+      getLockedPlanningSceneRO()->getCollisionEnv()->checkRobotCollision(collision_request, collision_result,
+                                                                         current_state);
+      scene_collision_distance = collision_result.distance;
+
+      collision_result.clear();
```

Comment:
We need to store the first bool collision result before clearing as well

Reply:
Good catch. Done